app/layouts/default.njk

```diff
@@ -1,9 +1,18 @@
 <!DOCTYPE html>
 <html class="{% block pageClass %}{% endblock %}" lang="en">
   <head>
+    <meta property="og:image" content="https://d33wubrfki0l68.cloudfront.net/8afee13b9e497a643b836562965a1a49e580e099/4bab0/images/cover.png">
```

Comment:
What is this image? It doesn't load for me.

Reply:
I'm not sure where this URL came from, but I think it should be

`/images/cover.png`

Reply:
yeah it should be that. its just the image that i want displayed when you share a link of the site.